index.bs

```diff
@@ -909,7 +972,49 @@ partial interface HTMLIFrameElement {
       <li>Return "<code>Disabled</code>".</li>
     </ol>
   </section>
+  <section>
+    <h3 id="report-violation">Generate report for violation of |feature| policy
+    with |message| and |disposition|, on |settings|</h3>
+
+    <p> Given a feature (|feature|), a string (|message|), another string
+    (|disposition|), an <a>environment settings object</a> (|settings|), and an
+    optional string (|group|), this algorithm generates a <a>report</a>,
+    containing |message|, about the <a>violation</a> of the policy on
+    |feature|.</p>
+
+    1.  Let |body| be a new {{FeaturePolicyViolationReportBody}}, initialized
+        as follows:
+
+        :   [=FeaturePolicyViolationReportBody/featureId=]
+        ::  |feature|'s string representation.
+        :   [=FeaturePolicyViolationReportBody/message=]
+        ::  |message|
```

Comment:
Message is not defined on FeaturePolicyViolationReportBody. I can't remember where we came down on whether a message was required or not, but the two sections here should match.